openapi_configuration.yml

```diff
@@ -919,6 +1077,248 @@ components:
       required:
         - versions
       additionalProperties: false
+    PostGoldenExtraction:
+      type: object
+      properties:
+        golden:
+          type: string
+          description: user-friendly name for the reference document #question: seems inconsistent that other parts of the internal API require doc type ID, not userfriendly name
+          example: best_scan_doc_num_1234
+        configuration:
+          type: string
+          description: SenseML configuration as stringified JSON. If you leave out this parameter, then Sensible returns the best-scoring extraction from the configurations in the document type.  #question: response doesn't seem to include classification summary, why not? 
+    
+    # NOTE: all the Extraction schema stuff is duplicated in openapi.yml it would be nicer to single source them but then we'd need a tool to combine the ymls since readme-cli doesn't seem to handle it well...
+    ResponseGoldenExtraction:
+      allOf:
+        - $ref: '#/components/schemas/Extraction'
+        - type: object
+          properties:
+            parsed_document_with_metadata:
+              type: object
+              description: the verbose extraction output, including information about line coordinates.
+            errors:
+              type: array
+              description: errors
+            text:  # question how/when is this optionally returned? I didn't see a param to request it in /extract_from_golden/{type}
```

Comment:
it should be driven by the `verbosity` setting in the configuration - a value of 3 will cause the text to be included

i'm not actually sure if this works offhand